cluster-install/install.sh

```diff
@@ -1253,7 +1337,7 @@ wait_for_cr_state() {
     local counter=0
     local current_state=""
     while ((counter < max_checks)); do
-        current_state=$(kubectl get -n "$namespace" "$resource" -o jsonpath="{$path}" 2> /dev/null)
+        current_state=$(k8s_get_resource "$resource" "$namespace" jsonpath="{$path}")
```

Comment:
As mentioned, you'll be seeing a lot of these since I replaced pretty much all `kubectl get` calls.